scripts/TDUIMod/ExtTDUIMod.py

```
@@ -48,6 +49,7 @@ def onButtonClick(self, panelValue):
 		if name == 'lselect':
 			op.VSN1_HOVER.openParameters()
 		elif name == 'rselect':
+			op.VNS1_HOVER.par.Enableui = True
```

Ruff (0.14.3)

Undefined name `op`

(F821)